server/server.go

```diff
@@ -219,25 +238,12 @@ func CreateServer(ctx context.Context, cfg *config.Config, apiBuilders ...Handle
 	if err != nil {
 		return nil, err
 	}
-	if len(apiBuilders) != 0 {
-		apiHandler, err := combineBuilderServerHTTPService(s, apiBuilders...)
+	if len(serviceBuilders) != 0 {
+		userHandlers, err := combineBuilderServerHTTPService(s, serviceBuilders...)
 		if err != nil {
 			return nil, err
 		}
-
-		etcdCfg.UserHandlers = map[string]http.Handler{
-			pdAPIPrefix: apiHandler,
-		}
-
-		if cfg.EnableDashboard {
-			etcdCfg.UserHandlers[dashboardUIPath] = http.StripPrefix(dashboardUIPath, uiserver.Handler())
-			etcdCfg.UserHandlers[dashboardAPIPath] = apiserver.Handler(dashboardAPIPath, &dashboardConfig.Config{
-				DataDir:    cfg.DataDir,
-				PDEndPoint: etcdCfg.ACUrls[0].String(),
-			})
-			log.Info("Enabled Dashboard API", zap.String("path", dashboardAPIPath))
-			log.Info("Enabled Dashboard UI", zap.String("path", dashboardUIPath))
-		}
+		etcdCfg.UserHandlers = userHandlers
```

Comment:
Do not use `EnableDashboard ` now? Is default open?

Reply:
mistake